openshift/scripts/oc_provision_matomo_backup.sh

```diff
@@ -1,5 +1,7 @@
 #!/bin/sh -l
 #
+# TODO: Delete this file! Superceded by oc_provision_backup_mariadb.sh in OCP4
```

Comment:
Is this file along with `oc_provision_matomo_backup_cronjob.sh` still needed?

Reply:
This file is needed until we cut over to silver. (This is currently running in OCP3).